iphone/Maps/Core/Theme/Extensions/UIColor+hexString.swift

```diff
@@ -0,0 +1,23 @@
+extension UIColor {
+    var hexString: String {
+      let cgColorInRGB = cgColor.converted(to: CGColorSpace(name: CGColorSpace.sRGB)!, intent: .defaultIntent, options: nil)!
+              let colorRef = cgColorInRGB.components
+              let r = colorRef?[0] ?? 0
+              let g = colorRef?[1] ?? 0
+              let b = ((colorRef?.count ?? 0) > 2 ? colorRef?[2] : g) ?? 0
```

Comment:
Is this step required after conversion to RGB color space above?

Reply:
sometimes the blue component is not presented, so we need to check for it and do fallback in this case